README.md

```diff
@@ -2,11 +2,61 @@
 
 # MakersBnB 🏠
 
-#### Technologies: Javascript, Jasmine, Ruby, Rspec, Capybara, Sinatra, ActiveRecord, HTML, CSS, RuboCop, ESLint
+#### Technologies: Javascript, Jasmine, Ruby, Rspec, Capybara, Sinatra, ActiveRecord, SQL, HTML, CSS, RuboCop, ESLint
 
-A web application that allows users to list spaces they have available, and to hire spaces for the night.
+[Task](#Task) | [Collaborators](#Collaborators) | [Installation Instructions](#Installation) | [Headline Specifications](#Headline_Specifications) | | [User Stories](#User_Stories) | [Domain Model](#Domain_Model) | [Mockup](#Mockup) | [Further improvements](#Further_Improvements)
 
-### Headline specifications
+![space-pic](space-pic.jpg)
+
+## <a name="Task">The Task</a>
+
+Build a web application that allows users to list spaces they have available, and to hire spaces for the night.
+
+This challenge is the first weekly team programming challenge at [Makers Academy](https://github.com/makersacademy).
+
+## <a name="Collaborators">Collaborators</a>
+
+[Elena Beccaro](https://github.com/elebecca)
+[Liz Daly](https://github.com/lookupdaily)
+[Barri Faryad](https://github.com/BarriF13)
+[Ellie Jones](https://github.com/EllieRichardsonJones)
+[David McGregor](https://github.com/davmcgregor)
+
+## <a name="Installation">Installation Instructions</a>
+
+1. Fork this repository, clone to your local machine then change into the directory:
+```
+$ git clone git@github.com:davmcgregor/Makersbnb.git
+$ cd Makersbnb
+```
+2. Load dependencies with bundle:
+```
+$ gem install bundle
+$ bundle
+```
+3. Access to the database is controlled by an ORM, ActiveRecord. ActiveRecord rake tasks will are used to migrate the development and test databases. 
+
+To migrate the development and test databases, please run the following command:
+```
+$ rake db:create
+```
+4. To poplulate the databases with the appropriate tables, run the following migrations:
+```
+$ rake db:migrate
+$ rake db:migrate RACK_ENV=test
+```
+5. Run the app on a local server: 
+```
+$ rackup
+```
+To view the app navigate to: http://localhost:9292/
+6. Tests were written with RSpec and Capybara. To run the tests in terminal: 
```

Comment:
Double check this line is on a new line?

Reply:
Thanks - fixed this